dashboards/MySQL_Replication_Summary.json

```
@@ -2067,16 +2266,26 @@
                 {
                     "content": "<h5><b><font color=#e68a00><center><a href=\"/graph/d/mysql-instance-summary/mysql-instance-summary?var-node_name=$node_name&var-service_name=$service_name\" target=\"_blank\">$crop_service...</a></center></font></b></h5>",
                     "datasource": null,
+                    "fieldConfig": {
+                        "defaults": {
+                            "custom": {}
+                        },
+                        "overrides": []
+                    },
                     "gridPos": {
                         "h": 2,
                         "w": 3,
                         "x": 21,
-                        "y": 38
+                        "y": 37
                     },
                     "id": 1000,
                     "links": [],
                     "mode": "html",
-                    "options": {},
+                    "options": {
+                        "content": "<h5><b><font color=#e68a00><center><a href=\"/graph/d/mysql-instance-summary/mysql-instance-summary?var-node_name=$node_name&var-service_name=$service_name\" target=\"_blank\">$crop_service...</a></center></font></b></h5>",
```

`<font>` is deprecated.
```suggestion
                        "content": "<h5><b><span style="color:#e68a00; text-align: center;"><a href=\"/graph/d/mysql-instance-summary/mysql-instance-summary?var-node_name=$node_name&var-service_name=$service_name\" target=\"_blank\">$crop_service...</a></span></b></h5>",
```

---

Thank you. I have missed this section.
It will be completely replaced in the ticket [PMM-6701](https://jira.percona.com/browse/PMM-6701)